core/sql/cli/sqlcli.h

```diff
@@ -1169,15 +1170,14 @@ enum SQLSTATS_ITEM_ID {
   SQLSTATS_BMO_HEAP_WM = 152,
   SQLSTATS_BMO_SPACE_BUFFER_SIZE = 153,
   SQLSTATS_BMO_SPACE_BUFFER_COUNT = 154,
-  SQLSTATS_OVEFLOW_PHASE_STARTTIME = 155,
-  SQLSTATS_OVERFLOW_PHASE = 156,
   SQLSTATS_SCRATCH_FILE_COUNT = 157,
   SQLSTATS_SCRATCH_BUFFER_BLOCK_SIZE = 158,
   SQLSTATS_SCRATCH_BUFFER_BLOCKS_READ = 159,
   SQLSTATS_SCRATCH_BUFFER_BLOCKS_WRITTEN = 160,
   SQLSTATS_SCRATCH_OVERFLOW_MODE = 161,
   SQLSTATS_SCRATCH_READ_COUNT = 162,
   SQLSTATS_SCRATCH_WRITE_COUNT = 163,
+  SQLSTATS_SCRATCH_IO_TIME = 155,
```

Comment:
Might be better to keep these in numeric order? I'm wondering what might happen if someone else added a new counter using 155, for example.